assets/wizards/popups/views/popup-group/index.js

```diff
@@ -19,89 +18,75 @@ import { find } from 'lodash';
 import { withWizardScreen, ActionCardSections } from '../../../../components/src';
 import PopupActionCard from '../../components/popup-action-card';
 
+const descriptionForPopup = (
+	{ categories, sitewide_default: sitewideDefault, options },
+	segments
+) => {
+	const segment = find( segments, [ 'id', options.selected_segment_id ] );
+	const descriptionMessages = [];
+	if ( segment ) {
+		descriptionMessages.push( `${ __( 'Segment:', 'newspack' ) } ${ segment.name }` );
+	}
+	if ( sitewideDefault ) {
+		descriptionMessages.push( __( 'Sitewide default', 'newspack' ) );
+	}
+	if ( options.placement === 'above_header' ) {
+		descriptionMessages.push( __( 'Above header', 'newspack' ) );
+	}
+	if ( categories.length > 0 ) {
+		descriptionMessages.push(
+			__( 'Categories: ', 'newspack' ) + categories.map( category => category.name ).join( ', ' )
+		);
+	}
+	return descriptionMessages.length ? descriptionMessages.join( ' | ' ) : null;
+};
+
 /**
  * Popup group screen
  */
+const PopupGroup = ( {
+	deletePopup,
+	emptyMessage,
+	items: { active = [], draft = [], test = [], inactive = [] },
+	previewPopup,
+	setCategoriesForPopup,
+	setSitewideDefaultPopup,
+	publishPopup,
+	updatePopup,
+	segments,
+} ) => {
+	const getCardClassName = ( { key }, { sitewide_default } ) =>
```

Comment:
Non-blocker, just a question: any reason this can't also be declared outside the component scope alongside `descriptionForPopup` since we're passing all the data as args?

Reply:
Good idea. Done in https://github.com/Automattic/newspack-plugin/pull/764/commits/756c02eff3cddb85fa0cd0452309b5a4b0b1e6ab. This function is actually removed in another PR, but there's a similar one added which should also be placed out side of the component scope.